vision/google/cloud/vision/_gax.py

```diff
@@ -14,12 +14,36 @@
 
 """GAX Client for interacting with the Google Cloud Vision API."""
 
+from google.cloud.gapic.vision.v1 import image_annotator_client
+from google.cloud.grpc.vision.v1 import image_annotator_pb2
+
+from google.cloud._helpers import _to_bytes
+
 
 class _GAPICVisionAPI(object):
     """Vision API for interacting with the gRPC version of Vision.
 
-    :type client: :class:`~google.cloud.core.client.Client`
+    :type client: :class:`~google.cloud.vision.client.Client`
     :param client: Instance of ``Client`` object.
     """
     def __init__(self, client=None):
-        raise NotImplementedError
+        self._client = client
+        self._api = image_annotator_client.ImageAnnotatorClient
+
+
+def _to_gapic_feature(feature):
+    """Helper function to convert a ``Feature`` to a gRPC ``Feature``."""
```

Comment:
Since they're private I didn't think that was needed since it's pretty self explanatory.

I'll add them to the next commit though.